tests/test_netdevices.py

```diff
@@ -208,6 +207,14 @@ def test_dump(self):
         output = out.getvalue()
         self.assertEqual(expected, output)
 
+    def test_os(self):
+        self.nd = NetDevices()
```

Comment:
You don't need to do this here since it's already being done in `TestNetDeviceObject.setUp()` (which is called before every test method is called).